scripts/rebuild-testbench.sh

```diff
@@ -34,22 +49,88 @@ export_CC_with_afl()
     export CC=${SOF_AFL}
 }
 
+setup_xtensa_tools_build()
+{
+    BUILD_TYPE=xt
+    BUILD_TARGET=
+    BUILD_DIR_NAME=build_xt_testbench
+
+    # check needed environment variables
+    test -z "${XTENSA_TOOLS_ROOT}" && die "XTENSA_TOOLS_ROOT need to be set."
+
+    # Get compiler version for platform
+    source "$SCRIPT_DIR/set_xtensa_params.sh" "$BUILD_PLATFORM"
+    test -z "${XTENSA_TOOLS_VERSION}" &&
+	die "Illegal platform $BUILD_PLATFORM, no XTENSA_TOOLS_VERSION found."
+    test -z "${XTENSA_CORE}" &&
+	die "Illegal platform $BUILD_PLATFORM, no XTENSA_CORE found."
```

Comment:
Nit:
```suggestion
	test -n ... || die "Illegal platform $BUILD_PLATFORM, no XTENSA_CORE found."
```

see above.